db.js

```diff
@@ -70,16 +69,14 @@ exports.init = function (sbot, dir, config) {
   }
 
   function get(id, cb) {
-    onIndexesStateLoaded(() => {
-      query(
-        and(key(id)),
-        toCallback((err, results) => {
-          if (err) return cb(err)
-          else if (results.length) return cb(null, results[0].value)
-          else return cb()
-        })
-      )
-    })
+    query(
```

Comment:
onIndexesStateLoaded was not needed because that's about base index, ebt index, private index, etc, and this query only uses JITDB things. It will use an implicit onDrain because it's `query`